Procurement/Controls/FragmentStashViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using POEApi.Model;
+using Procurement.ViewModel;
+
+namespace Procurement.Controls
+{
+    internal class FragmentStashViewModel
+    {
+        private readonly List<Item> _stash;
+
+        public FragmentStashViewModel(List<Item> stash)
+        {
+            _stash = stash;
+        }
+
+        public ItemDisplayViewModel Dawn => GetItemCalled("Sacrifice at Dawn");
+        public ItemDisplayViewModel Dusk => GetItemCalled("Sacrifice at Dusk");
+        public ItemDisplayViewModel Noon => GetItemCalled("Sacrifice at Noon");
+        public ItemDisplayViewModel Midnight => GetItemCalled("Sacrifice at Midnight");
+
+        public ItemDisplayViewModel Grief => GetItemCalled("Mortal Grief");
+        public ItemDisplayViewModel Rage => GetItemCalled("Mortal Rage");
+        public ItemDisplayViewModel Ignorance => GetItemCalled("Mortal Ignorance");
+        public ItemDisplayViewModel Hope => GetItemCalled("Mortal Hope");
+
+        public ItemDisplayViewModel Volkuur => GetItemCalled("Volkuur's Key");
+        public ItemDisplayViewModel Eber => GetItemCalled("Eber's Key");
+        public ItemDisplayViewModel Yriel => GetItemCalled("Yriel's Key");
+        public ItemDisplayViewModel Inya => GetItemCalled("Inya's Key");
+
+        public ItemDisplayViewModel Hydra => GetItemCalled("Fragment of the Hydra");
+        public ItemDisplayViewModel Phoenix => GetItemCalled("Fragment of the Phoenix");
+        public ItemDisplayViewModel Minotaur => GetItemCalled("Fragment of the Minotaur");
+        public ItemDisplayViewModel Chimera => GetItemCalled("Fragment of the Chimera");
+
+        public ItemDisplayViewModel Divine => GetItemCalled("Divine Vessel");
+
+        public ItemDisplayViewModel Chayula => new ItemDisplayViewModel(_stash.OfType<Splinter>().FirstOrDefault(x => x.Type == BreachType.Chayula));
```

Comment:
I'd recommend using more specific property names for the splinters, so they are easily distinguishable from the blessings.  For example, `ChayulaSplinter`.

Reply:
Done